Listing '.'...
Listing './bin'...
Listing './include'...
Listing './lib'...
Listing './lib/python3.5'...
Listing './lib/python3.5/site-packages'...
Listing './lib/python3.5/site-packages/pip'...
Listing './lib/python3.5/site-packages/pip/_vendor'...
Listing './lib/python3.5/site-packages/pip/commands'...
Listing './lib/python3.5/site-packages/pip/compat'...
Listing './lib/python3.5/site-packages/pip/models'...
Listing './lib/python3.5/site-packages/pip/operations'...
Listing './lib/python3.5/site-packages/pip/req'...
Listing './lib/python3.5/site-packages/pip/utils'...
Listing './lib/python3.5/site-packages/pip/vcs'...
Listing './lib/python3.5/site-packages/pip-9.0.1.dist-info'...
Listing './lib/python3.5/site-packages/pkg_resources'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor/packaging'...
Listing './lib/python3.5/site-packages/pkg_resources/extern'...
Listing './lib/python3.5/site-packages/pkg_resources-0.0.0.dist-info'...
Listing './lib/python3.5/site-packages/pyscanning'...
Listing './lib/python3.5/site-packages/pyscanning/backends'...
Compiling './lib/python3.5/site-packages/pyscanning/backends/_sane_.py'...
***   File "./lib/python3.5/site-packages/pyscanning/backends/_sane_.py", line 58
    def close(self):
      ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pyscanning/backends/_twain_.py'...
*** Sorry: TabError: inconsistent use of tabs and spaces in indentation (_twain_.py, line 9)
Listing './lib/python3.5/site-packages/pyscanning/testdata'...
Listing './lib/python3.5/site-packages/pyscanning-0.2.1.egg-info'...
Listing './lib/python3.5/site-packages/setuptools'...
Listing './lib/python3.5/site-packages/setuptools/command'...
Listing './lib/python3.5/site-packages/setuptools/extern'...
Listing './lib/python3.5/site-packages/setuptools-32.3.1.dist-info'...
Listing './share'...
Listing './share/python-wheels'...
